msal/mex.py

```diff
@@ -42,8 +42,7 @@ def _xpath_of_root(route_to_leaf):
 
 def send_request(mex_endpoint, http_client, **kwargs):
     mex_document = http_client.get(
-        mex_endpoint, headers={'Content-Type': 'application/soap+xml'},
-        **kwargs).text
+        mex_endpoint, **kwargs).text
```

Comment:
Might as well combine line 44 and 45 into one line, given that it is still shorter than 80 characters.